app/src/main/java/pl/gittobefit/running_training/fragment/TrainingStart.java

```diff
@@ -0,0 +1,396 @@
+    }
+
+    private void setupForCircuit()
+    { // zmiana na obwodowy
+        binding.scheduleTypeTitle.setText(getString(R.string.scheduleTypeCircuit));
+        binding.countOfSeries.setText(String.format("%s / %s",
+                model.getNumberOfSeries(),
+                model.getExerciseExecution().getSeries())
+        );
+    }
+
+    private void setupForSeries()
+    { // zmiana na seriowy
+        binding.scheduleTypeTitle.setText(getString(R.string.seriesTitle));
+        binding.countOfSeries.setText(String.valueOf(model.getExerciseExecution().getSeries()));
+    }
+
+    private void generateForegroundView()
+    { // wczytanie okienka po kliknieciu start
+        printDefault();
+        if(model.getExerciseExecution().getCount() == 0)
+        {
+            binding.textInfoNext.setText(getString(R.string.preview_info));
+            binding.timerPause.startAnimation(AnimationUtils.loadAnimation(getActivity(), R.anim.downtoup));
+            binding.textInfoNext.startAnimation(AnimationUtils.loadAnimation(getActivity(), R.anim.downtoup));
+            timerBeforeStartTraining();
+        }else
+        {
+            binding.textInfoNext.setText(getString(R.string.text_info_next));
+            binding.printCountOfRepeats.setText(String.valueOf(model.getExerciseExecution().getCount()));
+            binding.buttonClicker.setOnClickListener(v -> timerOnBreak());
+        }
+        if(model.getTrainingWithForm().form.getScheduleType().equals("CIRCUIT"))
+        {
+            binding.printScheduleTypeTitle.setText(getString(R.string.scheduleTypeCircuit));
+        }else
+        {
+            binding.printScheduleTypeTitle.setText(getString(R.string.scheduleTypeSeries));
+        }
+    }
+
+    private void printDefault()
+    {
+        binding.printCountOfSeries.setText(String.format("%s / %s",
+                model.getNumberOfSeries(),
+                model.getExerciseExecution().getSeries()
+                ));
+
+
+        if(model.getExerciseExecution().getCount() == 0)
+        {
+            binding.buttonClickButton.setVisibility(View.GONE);
+            binding.timerPause.setVisibility(View.VISIBLE);
+            binding.printCount.setVisibility(View.GONE);
+        }else
+        {
+            binding.buttonClickButton.setVisibility(View.VISIBLE);
+            binding.printCount.setVisibility(View.VISIBLE);
+            binding.timerPause.setVisibility(View.GONE);
+        }
+        binding.exerciseBackground.setVisibility(View.VISIBLE);
+        binding.exerciseStart.setVisibility(View.VISIBLE);
+        binding.exerciseStart.startAnimation(AnimationUtils.loadAnimation(getActivity(), R.anim.started));
+        getSmallVideo();
+        binding.printExerciseInfo.startAnimation(AnimationUtils.loadAnimation(getActivity(), R.anim.downtoup));
+        binding.printExerciseInfo.setVisibility(View.VISIBLE);
+    }
+
+    private void timerBreakDuringExercises()
+    {
+        new CountDownTimer(29000, 1000)
```

Comment:
Czas między przerwami będzie ustalany przez użytkownika. Nie jestem pewny ,ale  do tego czasu trzeba będzie dodawać 1 sekundę, gdyż nie będzie to responsywne z aplikacją (czas się rozpoczyna w momencie rozpoczęcia animacji).

new CountDownTimer(30000, 1000)